### DISCUSSION TITLE
Flex-Config deployment on Windows

### DISCUSSION THREAD

**Discussion by ksmith173** [twilio-professional-services/flex-project-template#146] [Q&A] [answered]
When running`npm run deploy:dev` via command prompt or powershell in Windows I get the following error:

`'ENVIRONMENT' is not recognized as an internal or external command, operable program or batch file.`

I assume I am doing something wrong here?

**Comment by Mauricio-Nocce** [comment]
I have the same problem, did you find a solution?

**Reply by ksmith173** [reply]
Yes, I had to add the following line to the .env file:
`ENVIRONMENT=dev`

And then I was able to run:
`node index.js`

**Comment by dremin** [comment, marked as answer]
Flex-config deployments no longer have this problem on Windows because the `ENVIRONMENT` variable has been moved to the `.env` file.
